frontend/src/utils/error.ts

```diff
@@ -0,0 +1,10 @@
+import type { ApiError } from "../client"
```

Comment:
Great work on the refactoring.  Can you please rename this file to something more specific.  "showToastOnError" or "handleError"

Reply:
Renamed file to `showToastOnError`.